lib/entities/request/ClassicRequest.ts

```diff
@@ -57,13 +57,14 @@ export class ClassicRequest implements QuoteRequest {
     });
   }
 
-  constructor(public readonly info: QuoteRequestInfo, public readonly config: ClassicConfig) {}
+  constructor(public readonly info: QuoteRequestInfo, public readonly config: ClassicConfig, public readonly source?: RequestSource) {}
```

Comment:
since `source` applies to all types of quotes from the interface, can we put it in the shared `QuoteRequestInfo` and parse it in `parseQuoteRequests`?

Reply:
Thanks for the comments. Fixed as you suggested.